website/pages/docs/internals/limits.mdx

```diff
@@ -58,9 +58,11 @@ stored separately, so the limit applies to each independently.
 Specifying distinct per-mount options, or using long mount point paths, can
 increase the space required per mount.
 
-The number of mount points can be monitored by reading the
-[`sys/auth`](/api-docs/system/audit) and
-[`sys/mounts`](/api-docs/system/mounts) endpoints, which contain all mount points in all namespaces.
+The number of mount points can be monitored by reading the 
+[`sys/auth`](/api-docs/system/audit) and [`sys/mounts`](/api-docs/system/mounts) 
```

Comment:
Huh, just noticed a pre-existing error in this bit: the link should be to api-docs/system/auth, not /audit.